packages/relay/src/lib/eth.ts

```diff
@@ -940,6 +940,11 @@ export class EthImpl implements Eth {
     const rSig = contractResult.r === null ? null : contractResult.r.substring(0, 66);
     const sSig = contractResult.s === null ? null : contractResult.s.substring(0, 66);
 
+    if (process.env.DEV_MODE && process.env.DEV_MODE === 'true' && contractResult.result === 'CONTRACT_REVERT_EXECUTED') {
```

Comment:
Add a `docs/dev-mode.md` file that highlights dev mode.
Under it add a specific section for ethersjs revert check that highlights the goal of this workaround

Reply:
Update the https://github.com/hashgraph/hedera-json-rpc-relay/tree/main/tools#supported-tools section with a note that points to the dev mode feature for ethersjs support